doc/modules/cassandra/pages/new/configuration.adoc

```diff
@@ -131,6 +130,9 @@ our parameters (incrementally, post https://issues.apache.org/jira/browse/CASSAN
 |enable_drop_compact_storage |drop_compact_storage_enabled |-
 |enable_user_defined_functions_threads |user_defined_functions_threads_enabled |-
 |enable_legacy_ssl_storage_port |legacy_ssl_storage_port_enabled |-
+|user_defined_function_fail_timeout |user_defined_functions_fail_timeout |ms
+|user_defined_function_warn_timeout |user_defined_functions_warn_timeout |ms
+|cache_load_timeout_seconds |cache_load_timeout |s
```

Comment:
Intentionally didn't add the others as we don't advertise them in docs as they are for advanced users